source/tutorials/atacama-soils.rst

```diff
@@ -134,22 +102,61 @@ generate and view a summary of how many sequences were obtained per sample.
      --m-barcodes-column barcode-sequence \
      --p-rev-comp-mapping-barcodes \
      --i-seqs emp-paired-end-sequences.qza \
-     --o-per-sample-sequences demux.qza \
+     --o-per-sample-sequences demux-full.qza \
      --o-error-correction-details demux-details.qza
 
+Let's subsample the data. We will perform this subsampling in this tutorial
+for two reasons - one, to speed up the tutorial run time, and two, to
+demonstrate the functionality.
+
+.. warning:: The following example of subsampling reads is meant to
```

Comment:
Love this. Do we want to be more explicit that e.g. "Subsampling reads is not a generally recommended approach to performing analyses"?

Reply:
Thanks! I don't agree with the idea that "subsampling reads is not a generally recommended approach to performing analyses" - its just not common in our analyses!